stable/prometheus-statsd-exporter/values.yaml

```diff
@@ -0,0 +1,31 @@
+# Default values for prometheus-statsd-exporter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: prom/statsd-exporter
+  tag: v0.7.0
+  pullPolicy: IfNotPresent
+
+service:
+  type: ClusterIP
+
+webPort: 9102
+udpPort: 9125
+tcpPort: 9125
+
+podAnnotations: {}
+
+priorityClassName: ""
+
+resources: {}
+
+nodeSelector: {}
+
+affinity: {}
+
+tolerations: []
+
+statsdMappingConfig: ""
```

Comment:
maybe give a reasonable default configuration here? For one, given there may be more than one exporter running, default to histograms for the timer metric type.

Reply:
Indeed, that it defaults to summary has historical reasons. If you're replacing statsd one-to-one, that makes sense, because you would only be running one of those really and it also precomputes quantiles. But in a more dynamic, cloud-native deployment, having more than one exporter process is useful and common.

You could even leave off everything but the default timer type, but I suppose a more extensive example gives people a better start for customization.